global.json

```diff
@@ -1,8 +1,8 @@
 {
   "sdk": {
-    "version": "3.1.100"
+    "version": "5.0.100"
```

Comment:
Configure a [`rollForward`](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#rollforward) policy. I'd use `latestFeature` or `latestMinor`:

> `latestFeature`: Uses the highest installed feature band and patch level that matches  the requested major and minor with a feature band and patch level that  is greater or equal than the specified value.  If not found, fails.
> `latestMinor`: Uses the highest installed feature band and patch level that matches the requested major and minor with a feature band and patch level that is greater or equal than the specified value.
If not found, fails.

Reply:
Set to latestFeature in 616b415c79cf54eab14a0fa3ae4bb5d3a2362cae